assets/js/partners/contact.js

```diff
@@ -0,0 +1,115 @@
+require('uswds');
```

Comment:
You shouldn't need to initialize `uswds` like this, since we inherit `uswds` through the Login.gov Design System.

To limit the size of the JavaScript, we only load the specific components we're using. If you there are other ones we need, can you add them to the [set of components in `main.js`](https://github.com/18F/identity-site/blob/c89f927244753aaf587108a9075920c6613e0fc3/assets/js/main.js) instead?

Reply:
Or do something similar to what we're doing in `main.js`, and initialize those components here, if those components are expected to be needed only on this contact page.

Reply:
great suggestion, thanks! I'll admit that was mostly heavy-handed to get the combo box working at the beginning but I never went back to refactor.

Reply:
I ended up adding them to `main.js` - I don't know if they will be used anywhere else but it felt like it would be less annoying to try and remember in the future. That said, @aduth, if you think it will significantly impact performance across the site I'm happy to move the initialization just to this page. Let me know!

Reply:
> I ended up adding them to `main.js` - I don't know if they will be used anywhere else but it felt like it would be less annoying to try and remember in the future. That said, @aduth, if you think it will significantly impact performance across the site I'm happy to move the initialization just to this page. Let me know!

I think that'll be fine 👍